.github/workflows/tex-pdf.yml

```diff
@@ -0,0 +1,72 @@
+# add .tex conversion and (and .pdf rendering of .tex file) to a pull request that adds/modifies a cgel tree.
+# currently assumes cgel file(s) in datasets/oneoff.
+
+name: Create tree .tex and .pdf 
+
+permissions:
+  contents: write
+  pull-requests: write
+  repository-projects: write
+
+on:
+  pull_request:
+    paths:
+      - datasets/oneoff/*.cgel
+
+jobs:
+  render:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      # https://github.com/Ana06/get-changed-files/releases/tag/v1.2
+      - uses: Ana06/get-changed-files@v2.3.0
```

Comment:
Is it recommended to use commands directly from other users' repos? I could imagine this being a security risk (in that somebody in the future could hack the user's account and substitute malicious code).

Reply:
That's a good point I hadn't considered. This project is forked from an action listed on the Actions Marketplace (https://github.com/jitterbit/get-changed-files). One disadvantage of using the original version is that it throws an error when the PR branch is behind commits of the branch being merged into. There's an optional flag to suppress errors so that the action can continue, but that seemed undesirable, so I switched to this repo. After looking into this more, it looks like both options carry appreciable security risks, which can be mitigated by adding a commit hash to the `- uses` tag (https://blog.gitguardian.com/github-actions-security-cheat-sheet/).

Reply:
There's a list of security-related best practices in the gitguardian article, which I'll make sure we're abiding by before I merge.